.ibm/pipelines/jobs/gke-helm.sh

```diff
@@ -16,7 +16,7 @@ handle_gke_helm() {
 
   K8S_CLUSTER_ROUTER_BASE=$GKE_INSTANCE_DOMAIN_NAME
   NAME_SPACE="showcase-k8s-ci-nightly"
-  NAME_SPACE_RBAC="showcase-rbac-k8s-ci-nightly"
+  export NAME_SPACE_RBAC="showcase-rbac-k8s-ci-nightly"
```

Comment:
A nitpick - `NAME_SPACE_RBAC` is exported just the line below. This is redundant to export it twice.
```suggestion
 NAME_SPACE_RBAC="showcase-rbac-k8s-ci-nightly"
```